object.go

```diff
@@ -19,6 +19,36 @@ type Object struct {
 	*Value
 }
 
+func (o *Object) MethodCall(methodName string, args ...Valuer) (*Value, error) {
+	ckey := C.CString(methodName)
+	defer C.free(unsafe.Pointer(ckey))
+
+	getRtn := C.ObjectGet(o.ptr, ckey)
+	err := getError(getRtn)
+	if err != nil {
+		return nil, err
+	}
+	fnVal := getValue(o.ctx, getRtn)
+
+	fn, err := fnVal.AsFunction()
+	if err != nil {
+		return nil, err
+	}
+
+	var argptr *C.ValuePtr
+	if len(args) > 0 {
+		var cArgs = make([]C.ValuePtr, len(args))
+		for i, arg := range args {
+			cArgs[i] = arg.value().ptr
+		}
+		argptr = (*C.ValuePtr)(unsafe.Pointer(&cArgs[0]))
+	}
+	fn.ctx.register()
+	rtn := C.FunctionCall(fn.ptr, o.ptr, C.int(len(args)), argptr)
+	fn.ctx.deregister()
+	return getValue(fn.ctx, rtn), getError(rtn)
```

Comment:
Lines 38-49 can be replaced with `fn.Call(obj, args...)` once the other PR is merged. I kept them separate since one is a breaking change.